vaadin-upload-flow-integration-tests/pom-bower-mode.xml

```diff
@@ -6,7 +6,7 @@
     <parent>
         <groupId>com.vaadin</groupId>
         <artifactId>vaadin-upload-flow-parent</artifactId>
-        <version>2.1-SNAPSHOT</version>
+        <version>2.2-SNAPSHOT</version>
```

Comment:
Nit: should we bump `icons` snapshot since there is a PR for that?

Reply:
Done.